controllers/novels.js

```diff
@@ -0,0 +1,38 @@
+const models = require('../models')
+
+const getAllNovels = async (request, response) => {
+  try {
+    const novels = await models.novels.findAll({ include: [{ model: models.authors }, { model: models.genres }] })
+
+    return response.send(novels)
+  } catch (error) {
+    return response.status(500).send('Unkown error while retrieving novels')
+  }
+}
+
+const getNovelByIdOrTitle = async (request, response) => {
+  try {
+    const { identifier } = (request.params)
+
+    const novel = await models.novels.findOne({
+      where: {
+        [models.Sequelize.Op.or]: [
+          { id: identifier },
+          { title: { [models.Sequelize.Op.like]: '%${ identifier }%' } }
```

Comment:
You've got single quotes here instead of backticks. That means it is always looking for a book with the title `%${ indetified }%` instead of subbing in what the user provided via string interpolation.